tests/subsys/mgmt/mcumgr/smp_version/boards/qemu_riscv64.conf

```diff
@@ -1 +1,2 @@
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_IGNORE_MIN=y
```

Comment:
why? The calculated value is 1024, and the stack is being set to 4096, so I'm not understanding why ignoring a warning which is only printed if 4096 is less then 1024 (which isn't the case here) is needed?